prow/plugins.yaml

```diff
@@ -141,12 +141,19 @@ slack:
     whitelist:
     - k8s-ci-robot
 
-milestone:
-  # You can curl the following endpoint in order to determine the github ID of your team
-  # responsible for maintaining the milestones:
-  # curl -H "Authorization: token <token>" https://api.github.com/orgs/<org-name>/teams
-  maintainers_id: 2460384
-  maintainers_team: kubernetes-milestone-maintainers
+repo_milestone:
+  # Default maintainer
+  '':
+    # You can curl the following endpoint in order to determine the github ID of your team
+    # responsible for maintaining the milestones:
+    # curl -H "Authorization: token <token>" https://api.github.com/orgs/<org-name>/teams
+    maintainers_id: 2460384
+    maintainers_team: kubernetes-milestone-maintainers
+
+  kubernetes/test-infra:
+    milestone:
+      maintainers_id: 2460384
+      maintainers_team: kubernetes-milestone-maintainers
```

Comment:
This is identical to the default, why are we specifying this here? If it should be here please remove the `milestone:` key like you did for the default.